boot/freeldr/freeldr/arch/realmode/int386.inc

```diff
@@ -28,28 +28,28 @@ Int386:
     popf
 
     /* Setup the registers */
-    mov ax, word ptr cs:[BSS_RegisterSet + REGS_DS]
+    mov ax, word ptr ss:[BSS_RegisterSet + REGS_DS]
```

Comment:
Why the choice of the stack segment for these? These variables are on the stack?

Reply:
Nope it's because the stack segment is always zero in this case.

Reply:
and the Code Segment is not zero!!!